src/messaging/ExchangeContext.cpp

```diff
@@ -689,5 +695,21 @@ void ExchangeContext::ExchangeSessionHolder::GrabExpiredSession(const SessionHan
     GrabUnchecked(session);
 }
 
+bool ExchangeContext::ShouldPeerBeActive()
```

Comment:
Suggest rename to `IsPeerLikelyActiveHint` or such.

Reply:
done